ibis/__init__.py

```diff
@@ -125,3 +128,18 @@ def connect(*args, **kwargs):
         setattr(proxy, name, getattr(backend, name))
 
     return proxy
+
+
+def __getattr__(name: str) -> Any:
+    if name == "NA":
+        warnings.warn(
+            "Accessing 'ibis.NA' is deprecated as of v9.1 and will be removed in a future version. "
+            "Use 'ibis.null()' instead.",
+            DeprecationWarning,
+            stacklevel=2,
+        )
+        import ibis
+
+        return ibis.null()
```

Comment:
I _think_ this should work instead, since `null` is already imported above:

```suggestion
        return null()
```

Reply:
Has it? I don't see an import of null.